Makefile

```diff
@@ -0,0 +1,134 @@
+## Haiku Generic Makefile ##
+
+## Fill in this file to specify the project being created, and the referenced
+## Makefile-Engine will do all of the hard work for you. This handles any
+## architecture of Haiku.
+
+# The name of the binary.
+NAME = SystemInfo
+
+# The type of binary, must be one of:
+#	APP:	Application
+#	SHARED:	Shared library or add-on
+#	STATIC:	Static library archive
+#	DRIVER: Kernel driver
+TYPE = APP
+
+# 	If you plan to use localization, specify the application's MIME signature.
+APP_MIME_SIG = 
+
+#	The following lines tell Pe and Eddie where the SRCS, RDEFS, and RSRCS are
+#	so that Pe and Eddie can fill them in for you.
+#%{
+# @src->@ 
+
+#	Specify the source files to use. Full paths or paths relative to the 
+#	Makefile can be included. All files, regardless of directory, will have
+#	their object files created in the common object directory. Note that this
+#	means this Makefile will not work correctly if two source files with the
+#	same name (source.c or source.cpp) are included from different directories.
+#	Also note that spaces in folder names do not work well with this Makefile.
+SRCS = \
+	 CPUPercent.cpp  \
+	 ImageView.cpp  \
+	 main.cpp  \
+	 MemUsage.cpp  \
+	 SIReplicant.cpp  \
+
+
+#	Specify the resource definition files to use. Full or relative paths can be
+#	used.
+RDEFS = \
+	 SystemInfo.rdef  \
+
+
+#	Specify the resource files to use. Full or relative paths can be used.
+#	Both RDEFS and RSRCS can be utilized in the same Makefile.
+RSRCS = \
+
+
+# End Pe/Eddie support.
+# @<-src@ 
+#%}
+
+#%}
+
+#	Specify libraries to link against.
+#	There are two acceptable forms of library specifications:
+#	-	if your library follows the naming pattern of libXXX.so or libXXX.a,
+#		you can simply specify XXX for the library. (e.g. the entry for
+#		"libtracker.so" would be "tracker")
+#
+#	-	for GCC-independent linking of standard C++ libraries, you can use
+#		$(STDCPPLIBS) instead of the raw "stdc++[.r4] [supc++]" library names.
+#
+#	- 	if your library does not follow the standard library naming scheme,
+#		you need to specify the path to the library and it's name.
+#		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
+LIBS =  /boot/system/lib/libbe.so /boot/system/lib/libroot.so /boot/system/lib/libstdc++.r4.so
```

Comment:
libroot is linked by default.
Using $(STDCPPLIBS) instead of libstdc++.r4.so makes it work for gcc2 and gcc5.
Looks better as: `LIBS = be $(STDCPPLIBS)`